wp-content/themes/core/blocks/tribe/query-results-count/style.pcss

```diff
@@ -9,6 +9,10 @@
 
 	@mixin t-body;
 
+	> p {
```

Comment:
🟠 It looks like the paragraph block in the Query No Results already has the margin top set to zero, so maybe we don't need this as a custom style?

If we do need it as a custom style, I'd rather see this applied w/ a custom class on the specific paragraph. It's possible some projects will have multiple paragraph blocks and we don't want to affect others

Reply:
This is the query results count block, which is different than the no results block. This block only has the one paragraph in it and should never have more than one. With every file named the same it gets confusing 😂